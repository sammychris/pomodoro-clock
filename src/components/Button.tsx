import React from "react";

const Button = ({ event, id, iClass }) => {
  return (
    <div className="container-buttons" onClick={event}>
      <i id={id} className={iClass}></i>
    </div>
  );
};

export default Button;
