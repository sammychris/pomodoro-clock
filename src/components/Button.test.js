import { shallow } from "enzyme";
import Button from "./Button";

const wrapper = shallow(<Button />);

test("expect to render a button", () => {
  expect(wrapper.length).toEqual(1);
});

test("expect to make a snap shot to a component", () => {
  expect(wrapper).toMatchSnapshot();
});
