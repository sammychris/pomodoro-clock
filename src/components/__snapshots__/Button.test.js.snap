// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to make a snap shot to a component 1`] = `ShallowWrapper {}`;
